app_dart/lib/src/request_handlers/flaky_handler_utils.dart

```diff
@@ -74,11 +85,16 @@ Please follow https://github.com/flutter/flutter/wiki/Reducing-Test-Flakiness#fi
   }
 
   List<String> get issueLabels {
-    return <String>[
+    final List<String> labels = <String>[
       kTeamFlakeLabel,
       kSevereFlakeLabel,
       kP1Label,
     ];
+    final String teamLabel = _getTeamLabelFromTeam(ownership.team);
+    if (teamLabel?.isNotEmpty == true) {
```

Comment:
Nit: skip `== true`.

Reply:
is
```dart
if (teamLabel != null && teamLabel!. isNotEmpty)
```
prefer over this?

In flutter/flutter we usually do this instead

Reply:
TIL. Let's keep it with flutter/flutter then.